camayoc/tests/qpc/cli/test_scans.py

```diff
@@ -182,6 +182,77 @@ def test_create_scan_with_options(isolated_filesystem,
     scan_show_and_check(scan_name, expected_result)
 
 
+def test_create_scan_with_disabled_products_negative(isolated_filesystem,
+                                                     qpc_server_config,
+                                                     source):
+    """Attempt to create a scan with invalid values for disabled products.
+
+    :id: 4acaaeb4-3833-11e8-b467-0ed5f89f718b
+    :description: Create a scan with invalid values for disabled products.
+    :steps:
+        1) Call scan_add_and_check which runs the command
+           ``qpc scan add --sources <source> --disabled-optional-products
+           <optional-product>`` and asserts that it fails
+    :expectedresults: The scan is not created
+    """
+    scan_name = uuid4()
+    source_name = config_sources()[0]['name']
+    scan_add_and_check({
+        'name': scan_name,
+        'sources': source_name,
+        'disabled-optional-products': 'redhat_packages',
```

Comment:
Could parametrize the test to test a few bad values. Maybe include some names of facts, integers, dictionaries.

On the top of the test declaration add:
```
@pytest.mark.parametrize('one_item',  ITERABLE_THING)
def test_my_doodad(one_item):
      assert one_item == 100
```
https://docs.pytest.org/en/latest/parametrize.html#parametrize-basics